ConsentViewController/Classes/SPError.swift

```diff
@@ -315,6 +315,26 @@ import Foundation
     override public var spCode: String { "sp_metric_error_converting_consent_snapshot_to_json" }
 }
 
-@objcMembers public class InvalidJSONEncodeResult: SPError {
-    override public var spCode: String { "sp_metric_error_invalid_JSON_encode_result" }
+public class NetworkCallErrorsCode {
```

Comment:
Could we use the `enum InvalidResponsAPICode` only and get rid of the class?

I believe the following should work:
```swift
enum InvalidResponsAPI: String {
    case META_DATA = "_meta-data"
    case CONSENT_STATUS = "_consent-status"
    case PV_DATA = "_pv-data"
    case MESSAGES = "_messages"
    case EMPTY = ""

    var code: String {
        "sp_metric_invalid_response_api\(rawValue)"
    }
}
```